src/or/dirauth/dirvote.h

```diff
@@ -113,6 +115,13 @@ dirvote_free_all(void)
 {
 }
 
+static inline void
+dirvote_parse_sr_commits(networkstatus_t *ns, smartlist_t *tokens)
```

Comment:
Why should anything be calling this method if the module is disabled?

Reply:
Should not! But it is used in `routerparse.c` within `networkstatus_parse_vote_from_string()` which parses ALL THE THINGS including votes which are dirauth only.

So this is one of the example that once we can extract the vote parsing in some ways into the dirauth module, this function won't be public anymore.